data/channels.csv

```diff
@@ -18890,7 +18890,7 @@ ProfessionalBaseball4.kr,Professional Baseball 4,프로야구 4,,,KR,,,c/KR,kor,
 ProfessionalBaseball5.kr,Professional Baseball 5,프로야구 5,,,KR,,,c/KR,kor,sports,FALSE,,,,,https://i.imgur.com/vfrecYS.png
 ProfitTV.ro,Profit TV,,,,RO,,,c/RO,ron,,FALSE,,,,https://www.profit.ro/,https://i.imgur.com/uPmPAED.png
 ProLyubov.ru,Pro Lyubov,Про Любовь,,,RU,,,c/RU,rus,,FALSE,,,,https://www.tricolor.tv/channels/pro-lyubov/,https://i.imgur.com/QpIGAT0.png
-PromarTV.ve,PromarTV,,,,VE,,Yaracuy,c/VE,spa,general,FALSE,1995-11-01,,,https://www.promar.tv/,https://upload.wikimedia.org/wikipedia/commons/9/9f/Promar_Television.png

```

Comment:
Why did you remove the link to the site and the launch date?

Reply:
So long ago dont remember: amend as you see fit.